### ISSUE TITLE
FileUpload should support Accessibility guidelines

### ISSUE THREAD

**Issue by ambivorous** [primefaces/primeng#11620]
### Describe the feature you would like to see added

There is no way to assign an ID to the file input, nor are there any assignable aria attributes for screen readers.

### Is your feature request related to a problem?

FileUpload inputs are not associated with their form labels. This violates WCAG 2.1, 4.1.2 Name, Role, Value (Level A).

### Describe the solution you'd like

I would like to see attributes like inputId and/or ariaLabelledBy added to the FileUpload component as exists on other components.

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,

**Comment by Ivanho92** [comment]
Can we reopen this issue? This has not been implemented yet in version 17.

Thanks.
